lua/modules/hyperspace.i

```diff
@@ -1471,6 +1471,301 @@ playerVariableType playerVariables;
 %rename("%s") Projectile::flashTracker;
 %rename("%s") Projectile::color;
 
+
+%nodefaultctor CrewMember;
+%nodefaultdtor CrewMember;
+%rename("%s") CrewMember;
+
+%rename("%s") Get_CrewMember_Extend;
```

Comment:
One thing we wanted to do, or at least mathchamp expressed something like it, and this might require more changes on the C/C++ side, is that ideally we'd simplify our "Extended" crewmember into just a "CrewMember" object in Lua. Maybe by adding things to the CrewMember metatable or whatever but hopefully we can find a way of exposing the CrewMember_Extend items as part of regular CrewMember (or a differently named proxy class)